lib/active_merchant/lib/connection.rb

```diff
@@ -95,41 +95,45 @@ def http
       configure_cert(http)
       http
     end
-    
+
     def configure_debugging(http)
       http.set_debug_output(wiredump_device)
     end
-    
+
     def configure_timeouts(http)
       http.open_timeout = open_timeout
       http.read_timeout = read_timeout
     end
-    
+
     def configure_ssl(http)
       return unless endpoint.scheme == "https"
 
       http.use_ssl = true
-      
+
+      http.instance_eval do
+        @ssl_context.set_params(:options => OpenSSL::SSL::OP_NO_SSLv3 + OpenSSL::SSL::OP_NO_SSLv2)
```

Comment:
@TaylorChristie this actually fixed it. 🤷‍♂️

Reply:
@jasonewall  wow well, at least it works now. Did we revert the verify_mode?

Reply:
Yes